nzedb/Binaries.php

```diff
@@ -637,14 +637,20 @@ public function scan($groupMySQL, $first, $last, $type = 'update', $missingParts
 		// If there was an error, try to reconnect.
 		if ($this->_nntp->isError($headers)) {
 
+			if ($this->first == $this->last) {
```

Comment:
Doesn't really matter.  The whole point of doing the ternary in the sprintf (I believe) in the first place was to save memory as part of the optimizations @kevinlekiller did when he tried to optimize binary updates as much as possible.  So if the requirement is to set it to a var for sprintf, it defeats the purpose of using the ternary which is much more annoying to look at than an if/then/else.